cli/src/main.rs

```diff
@@ -178,6 +185,16 @@ fn get_default_keypair_path() -> PathBuf {
     path
 }
 
+fn print_output<Output: fmt::Display + Serialize>(as_json: bool, output: &Output) {
```

Comment:
Maybe a save to file as well (in the future)? I find it annoying to read json output in the terminal, even with `jq`

Reply:
You can redirect stdout to a file if you want?

```console
$ multisig --foobar --output-json > output.json
```